narf/combineutils.py

```diff
@@ -386,6 +400,18 @@ def __init__(self, indata, options):
         # observed number of events per bin
         self.nobs = tf.Variable(self.indata.data_obs, trainable=False, name="nobs")
 
+        if self.chisqFit:
+            if self.externalCovariance:
+                if self.indata.data_cov_inv is None:
+                    raise RuntimeError("No external covariance found in input data.")
+                # provided covariance
+                self.data_cov_inv = tf.Variable(self.indata.data_cov_inv, trainable=False, name="data_cov_inv")
```

Comment:
why does this need to be a tf.Variable rather than using the underlying tensor directly?
(only benefit to using a tf.Variable is if the value needs to be changed later which I don't think is the case)

Reply:
Done